script/Arcueid/Day 9/09_00_ARC09_1.txt

```diff
@@ -136,8 +137,9 @@ A child lies crumpled in the clearing.
 [sha:9b13c7cf5b90dbcd5fdd4990fe9f41bdac9d2939]{
 -- Page 5, Offset 21319.
 -- 白いシャツが真っ赤に染まって、ぴくりとも動かない。
-A deep red stain has blossomed on its white shirt.
- It doesn't move so much as an eyebrow.
+// Any reason for "it?" "They" seems more appropriate for a human being.
```

Comment:
There are a couple places in the text where the corpse is referred to in a very inanimate fashion, but this isn't really one of them. This sentence doesn't actually have a direct reference to the child, so we can go with they/them.